src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -328,6 +329,67 @@ internal static void SubscribeSearchForPackagesOption(Step stepInfo, StepUIAutom
                 foundUIElement.Click -= SearchForPackage_Click;
         }
 
+        /// <summary>
+        /// This method will subscribe the Next button from the Step8 Popup for clicking the Package already installed (then it will be expanded).
+        /// </summary>
+        /// <param name="stepInfo">Information about the Step</param>
+        /// <param name="uiAutomationData">Specific UI Automation step that is being executed</param>
+        /// <param name="enableFunction">it says if the functionality should be enabled or disabled</param>
+        /// <param name="currentFlow">The current flow of the Guide can be FORWARD or BACKWARD</param>
+        internal static void SubscribeNextButtonClickEvent(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            var nextbuttonFound = Guide.FindChild((stepInfo.StepUIPopup as PopupWindow).mainPopupGrid, "NextButton") as Button;
+            if (nextbuttonFound == null) return;
+            if(enableFunction)
+                nextbuttonFound.Click += ExecuteAutomaticPackage_Click;
+            else
+                nextbuttonFound.Click -= ExecuteAutomaticPackage_Click;
+                
+        }
+
+        /// <summary>
+        /// This method will execute the InvokeScript method using reflection so the overlay will be shown over the LibraryView.Browser
+        /// </summary>
+        /// <param name="stepInfo">Information about the Step</param>
+        /// <param name="uiAutomationData">Specific UI Automation step that is being executed</param>
+        /// <param name="enableFunction">it says if the functionality should be enabled or disabled</param>
+        /// <param name="currentFlow">The current flow of the Guide can be FORWARD or BACKWARD</param>
+        internal static void ShowOverlayInWebBrowser(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
```

Comment:
Isn`t it possible to merge this method with `SubscribePackageClicked`? Seems like both are doing the same thing except for the number of parameters.

Reply:
@filipeotero I've removed this method from the GuidesValidationMethods.cs class and now the info is in the json, also there is a section that will execute the js function, in this way we can reuse the code for setting the parameters and executing the js function. See the implementation in the next commit: 
https://github.com/DynamoDS/Dynamo/pull/12312/commits/2933fbf31bc87989ba49d35636b97c904c986d5f
Thanks